shared/src/main/scala/com/cibo/evilplot/plot/components/GenericDrawable.scala

```diff
@@ -0,0 +1,33 @@
+package com.cibo.evilplot.plot.components
+
+import com.cibo.evilplot.geometry.{Drawable, Extent}
+import com.cibo.evilplot.plot.Plot
+import com.cibo.evilplot.plot.aesthetics.Theme
+
+case class GenericDrawable(
+  position: Position,
+  f: (Extent => Drawable),
```

Comment:
I'm a bit confused here re: the `Extent => Drawable` that gets passed in along with another `Extent`. are we feeding the plot's size to the function in order to render it? in the examples I'm seeing a lot of `_ => Rect(25)` or similar, what's the use case for actually using that extent parameter / can it just take a `Drawable`?

Reply:
This bit was pulled from the axis marker work I was doing. In that case I modeled it after existing components that rendered drawables. My guess is that this was so drawables (in pixel) can be scaled based on the size of the plot. 🤷‍♂️